Make failing to load the Java ATK wrapper non-fatal

---
 src/java.desktop/share/classes/java/awt/Toolkit.java |    4 ++++
 1 file changed, 4 insertions(+)

--- a/src/java.desktop/share/classes/java/awt/Toolkit.java
+++ b/src/java.desktop/share/classes/java/awt/Toolkit.java
@@ -610,7 +610,11 @@ public abstract class Toolkit {
                 }
             });
             if (!GraphicsEnvironment.isHeadless()) {
+              try {
                 loadAssistiveTechnologies();
+              } catch (java.awt.AWTError e) {
+                // too bad
+              }
             }
         }
         return toolkit;
